cloud/observability/promql-to-scrape/examples/config.yaml

```diff
@@ -1,43 +1,43 @@
 metrics:
-    - metric_name: temporal_cloud_v0_frontend_service_error_count:rate1m
+    - metric_name: temporal_cloud_v0_frontend_service_error_count
       query: rate(temporal_cloud_v0_frontend_service_error_count[1m])
     - metric_name: temporal_cloud_v0_frontend_service_pending_requests
       query: temporal_cloud_v0_frontend_service_pending_requests
-    - metric_name: temporal_cloud_v0_frontend_service_request_count:rate1m
+    - metric_name: temporal_cloud_v0_frontend_service_request_count
       query: rate(temporal_cloud_v0_frontend_service_request_count[1m])
```

Comment:
The simple metric name `temporal_cloud_v0_frontend_service_request_count` generally implies raw counter data, but the query `rate(temporal_cloud_v0_frontend_service_request_count[1m])` actually computes a per-minute rate. This mismatch would create confusion for users who adopt this configuration.

Consider either:
1. Removing the rate calculation to match the metric name, or 
2. Keep following Prometheus conventions by not removing `:rate1m` to indicate this is a derived metric

Users rely on this as their default configuration, so alignment between metric names and their underlying calculations is important.

Reply:
sure! that makes sense. I was going off of the config your config https://gist.github.com/srikanthccv/b8ee857b7289a2ed29ca136fd037e4a0 in which you removed it. It's prob better left to the user to determine what they want to export it as.

FYI, [we have an update to the docs](https://github.com/temporalio/documentation/pull/3264) about the resolution of metrics:
> Metrics are scraped every 30s, which gets exposed to the metrics endpoint with a 1 min lag.
The endpoint has returns a resolution of 15s, which results in showing the same value twice.